activation/src/main/java/jakarta/activation/MimetypesFileTypeMap.java

```diff
@@ -238,13 +250,18 @@ private void loadAllResources(Vector v, String name) {
     /**
      * Load the named file.
      */
-    private MimeTypeFile loadFile(String name) {
-	MimeTypeFile mtf = null;
+    private MimeTypeRegistry loadFile(String name) {
+	MimeTypeRegistry mtf = null;
 
 	try {
-	    mtf = new MimeTypeFile(name);
+	    mtf = ServiceLoader.load(MimeTypeRegistryProvider.class).iterator().next().getByFileName(name);
 	} catch (IOException e) {
 	    //	e.printStackTrace();
```

Comment:
do not swallow exceptions. Either remove the catch or move it to the next one so it is at least logged